tasks/minsize/minsize.snap

```diff
@@ -11,15 +11,15 @@ Original   -> Minified   -> Gzip       Brotli
 
 555.77 kB  -> 274.89 kB  -> 91.40 kB   76.86 kB   d3.js
 
-977.19 kB  -> 456.43 kB  -> 123.71 kB  107.19 kB  bundle.min.js
+977.19 kB  -> 456.42 kB  -> 123.70 kB  107.22 kB  bundle.min.js
 
-1.25 MB    -> 677.73 kB  -> 166.57 kB  134.53 kB  three.js
+1.25 MB    -> 677.71 kB  -> 166.56 kB  134.62 kB  three.js
 
-2.14 MB    -> 743.85 kB  -> 181.82 kB  138.77 kB  victory.js
+2.14 MB    -> 743.83 kB  -> 181.81 kB  138.96 kB  victory.js
 
 3.20 MB    -> 1.03 MB    -> 332.92 kB  268.96 kB  echarts.js
 
-6.69 MB    -> 2.40 MB    -> 498.18 kB  390.19 kB  antd.js
```

Comment:
`drop_console` isn't enabled by default, so all that's going on here is that undefined initialization are getting stripped. Why would this make brotli compression worse?